docs/releases/2.1.txt

```diff
@@ -157,6 +157,15 @@ Models
 
 * Models can now use ``__init_subclass__()`` from :pep:`487`.
 
+* Added the :class:~django.db.models.functions.Ord database function.
```

Comment:
You need to add ticks around `~django.db.models.functions.Ord`. Perhaps these four could be merged into a single paragraph?